genome_browser/_genome_diagram.py

```diff
@@ -10,11 +9,11 @@
 
 
 class GenomeDiagram(object):
-    def __init__(self, targets, name=None):
-        self.targets = [targets] if not isinstance(targets, (list, tuple)) else targets
+    def __init__(self, name=""):
         self.name = name
+        self.annotation = ""
 
-        self.annotation = None
+        self.targets = []
```

Comment:
`targets` now work symmetrically to `tracks`.

Cause: The API on the `cv_update_api` branch felt unintuitive compared to `master`: one needed to always pass an additional argument.